src/api/spec/controllers/cloud/upload_jobs_controller_spec.rb

```diff
@@ -47,14 +47,16 @@
         HEREDOC
       end
 
+      before do
+        allow(Feature).to receive(:active?).with(:cloud_upload).and_return(true)
```

Comment:
Nitpick: This `allow` sentence, that returns `true`, is repeated three more times later: lines 51, 94, 147 and 192. They can be removed if we add this line in the parent `before` section, after line 36.